### ISSUE TITLE
API changed since last release

### ISSUE THREAD

**Issue by ghost** [fxwiegand/terrarium-stand#3]
You may want to upgrade `DaisySP` and `libDaisy` to their latest versions, there are a lot of new features and functions available. When upgrading to `master` for both branches all that really needs changing is the callback buffer. Instead of a float buffer it uses a type of `AudioHandle::InterleavingXXXXXBuffer` which are not pointers.

```
static void callback(
    AudioHandle::InterleavingInputBuffer  in,
    AudioHandle::InterleavingOutputBuffer out,
    size_t size
)
```

**Comment by fxwiegand** [comment]
I would be happy to merge a PR if you want to make the mentioned changes 😊

**Comment by ghost** [comment]
PR #4 👍🏻

**Comment by fxwiegand** [comment]
Just merged #4! Thanks a lot for the contribution! I hope this will help people to get startet in developing.
